OverlayPlugin.Core/ClipboardTechSupport.cs

```diff
@@ -0,0 +1,156 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Windows.Forms;
+using Advanced_Combat_Tracker;
+using Newtonsoft.Json.Linq;
+using RainbowMage.OverlayPlugin.EventSources;
+
+// TODO: print warnings on plugin ordering
+// TODO: get Ravahn to expose more settings and include them
+// TODO: print warning on cactbot plugin / url / user dir mismatch
+// TODO: include first N lines of OverlayPlugin log
+
+namespace RainbowMage.OverlayPlugin
+{
+    using SimpleTable = List<List<string>>;
+
+    // Facilitates copying useful info to the clipboard about OP plugins and overlays
+    // so that folks don't have to paste screenshots in discord.
+    class ClipboardTechSupport
+    {
+        private SimpleTable plugins;
+        private SimpleTable overlays;
+        private SimpleTable settings;
+        private SimpleTable warnings;
+
+        public ClipboardTechSupport(TinyIoCContainer container)
+        {
+            var repository = container.Resolve<FFXIVRepository>();
+
+            warnings = new SimpleTable { new List<string> { "Warnings" } };
+            var opVersion = repository.GetOverlayPluginVersion();
+            if (opVersion < new Version(0, 19, 0))
+            {
+                warnings.Add(new List<string> { $"Out of date OverlayPlugin {opVersion.ToString()}, see https://overlayplugin.github.io/OverlayPlugin/fork_update.html" });
+            }
```

Comment:
This check makes no sense in context. It's only going to exist on 0.19.0+ versions.

Reply:
Haha, this is what I get for trying to write code when I am really tired.  Oops, sorry.